index.html

```diff
@@ -27,6 +27,139 @@
+                        <p>A daily selection of privately </br> personalized reads; no accounts or </br> sign-ups required. has been the </br> industry's standard</p>
+                    </div>
+
+                    <div class="siklls skills2">
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
+                    <button class="project-2-btn">See Project</button>
+                </div>
+                <div class="project-2">
+                    <div class="project-2-text">
+                        <h3>Profesional Art </br> Printing Data</h3>
+                        <p>A daily selection of privately </br> personalized reads; no accounts or </br> sign-ups required. has been the </br> industry's standard</p>
+                    </div>
+
+                    <div class="siklls skills2">
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
+                    <button class="project-2-btn">See Project</button>
+                </div>
+                <div class="project-2">
+                    <div class="project-2-text">
+                        <h3>Profesional Art </br> Printing Data</h3>
+                        <p>A daily selection of privately </br> personalized reads; no accounts or </br> sign-ups required. has been the </br> industry's standard</p>
+                    </div>
+
+                    <div class="siklls skills2">
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
+                    <button class="project-2-btn">See Project</button>
+                </div>
+                <div class="project-2">
+                    <div class="project-2-text">
+                        <h3>Profesional Art </br> Printing Data</h3>
+                        <p>A daily selection of privately </br> personalized reads; no accounts or </br> sign-ups required. has been the </br> industry's standard</p>
+                    </div>
+
+                    <div class="siklls skills2">
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
+                    <button class="project-2-btn">See Project</button>
+                </div>
+            </div>
+        </section>
+        <section class="about-myself">
+            <div class="aboutme">
+                <h1 class="about-h1">About me</h1>
+                <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button class="about-btn">Get My Resume</button>
+                <hr>
+            </div>
```

Comment:
- [ ] Kindly ensure the `Get My Resume button` follows the design like the Figma template. Below is the image of your page on screen width: `375px - 445px`.
**Also, remember to add the spacing at the bottom of the button.**

| Figma design | Your Design |
| --------------- | ------------------- |
| ![Screenshot from 2022-09-24 12-48-24](https://user-images.githubusercontent.com/83042742/192093892-f05ba571-e893-4251-ada2-6bf1486b05ff.png) | ![Screenshot from 2022-09-24 12-47-04](https://user-images.githubusercontent.com/83042742/192093898-38088443-07a7-4344-bae2-68592b490229.png) |